ide/che-ide-gwt-app/src/main/resources/org/eclipse/che/ide/public/IDE.html

```diff
@@ -56,6 +56,81 @@
 
         var Loader = new function() {
 
+            /*
+             * Load keycloak settings
+             */
+            this.loadKeycloakSettings = function() {
+                var msg = "Cannot load keycloak settings";
```

Comment:
Please add explanation in message that this is normal for single-user mode.

Reply:
Done